setup_creators/get_tutorial.sh

```
@@ -1,15 +1,11 @@
 #!/bin/bash
 
-# 1. with this command we ensure that
-#    we are working in your HOME directory:
-cd
-
-# 2. we are going to work in a dedicated directory:
+# 1. we are going to work in a dedicated directory:
 mkdir snakemake-tutorial && cd snakemake-tutorial
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)